tests/ui/invalid_pyclass_args.rs

```diff
@@ -76,4 +76,13 @@ struct InvalidOrderedStruct {
     inner: i32
 }
 
+#[pyclass]
+struct Example {
+    #[pyo3(foo)]
+    #[pyo3(blah)]
+    x: i32,
+    #[pyo3(pop)]
+    y: i32,
+}
```

Comment:
Let's call this something more descriptive:

```suggestion
#[pyclass]
struct MultipleErrors {
    #[pyo3(foo)]
    #[pyo3(blah)]
    x: i32,
    #[pyo3(pop)]
    y: i32,
}
```